src/browser/api_protocol/api_handlers/system.ts

```diff
@@ -509,7 +509,7 @@ function downloadPreloadScripts(identity: Identity, message: APIMessage, ack: Ac
     const { payload: { scripts } } = message;
 
     System.downloadPreloadScripts(identity, scripts)
-        .then((downloadResults) => {
+        .then((downloadResults: any) => {
```

Comment:
Please, use download results' shape we already have here: https://github.com/HadoukenIO/core/blob/develop/src/browser/preload_scripts.ts#L14

Reply:
DownloadResult is not exported yet. Should we change it?